input-explainer.md

```diff
@@ -342,7 +342,7 @@ interface XRInputSource {
 interface XRInputPose {
   readonly attribute boolean emulatedPosition;
   readonly attribute XRRay targetRay;
-  readonly attribute Float32Array? gripMatrix;
+  readonly attribute XRRigidTransform? grip;
```

Comment:
`gripTransform` for better symmetry with `targetRay`?  This also makes it consistent that every attribute returning you a transform has `transform` in the name.